common/app/common/editions/Europe.scala

```diff
@@ -0,0 +1,20 @@
+package common.editions
+
+import java.util.Locale
+import common._
+import navigation.{EditionNavLinks, NavLinks}
+import org.joda.time.DateTimeZone
+
+object Europe
+    extends Edition(
+      id = "EUR",
+      displayName = "Europe edition",
+      timezone = DateTimeZone.forID("Europe/London"),
+      locale = Locale.forLanguageTag("en"), // #To-do: change this if we decide to include Europe in page alternates
```

Comment:
Would it make sense to change the `Locale` field to be optional?

Also - how does this PR exclude Europe from the hreflang alternate pages?